setup.py

```diff
@@ -45,7 +45,8 @@
             'data/models/t2/*/*/variables*',
             'data/catalogs/*.parquet',
             'data/catalogs/*.fits',
-            'data/catalogs/*.csv'],
+            'data/catalogs/*.csv',
+            'data/alerts/datatest/*.parquet'],
```

Comment:
The parquet files used in the test suite should not be packaged. You can remove the line you've added.